src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/bpp.jelly

```diff
@@ -365,19 +367,23 @@
 									<!-- Only adding the revision-cell once. -->
 									<j:if test="${x == 0}"><td class="revision-cell" rowspan="${buildGrid.rows}">
 										<table class="revision rounded build-card">
-											<tbody>
-												<tr class="header">
```

Comment:
What was the purpose of moving/removing existing classes? If somebody has a custom CSS theme, this change will break them